docs/operations/automated_backup_deletion_and_retention_de.md

```diff
@@ -0,0 +1,43 @@
+# Automatisierte Löschung und Aufbewahrung von Backups
+
+Backups können automatisch gelöscht werden.
+Standardmäßig werden keine Backups gelöscht.
+
+Um zu steuern, welche Backups gelöscht werden, kann eine von mehreren Aufbewahrungsstrategien aktiviert werden.
+Die Strategie kann in der `k8s-backup-operator-retention` ConfigMap konfiguriert werden:
+Automatisches Löschen und die Aufbewahrung können über die Values der Komponente konfiguriert werden.
+
+```yaml backup-operator-component.yaml
+apiVersion: k8s.cloudogu.com/v1
+kind: Component
+metadata:
+  name: k8s-backup-operator
+spec:
+  name: k8s-backup-operator
+  namespace: k8s
+  deployNamespace: ecosystem
+  valuesYamlOverwrite: |
+    retention:
+      # Die Strategie mit welcher Backups gelöscht werden. Standardmäßig is es keepAll.
+      strategy: keepLastSevenDays
+      # Dieses Cron-Pattern definiert, wie oft Backups gelöscht werden sollen.
+      # Standardeinstellung ist "0 * * * *", also jede volle Stunde.
+      garbageCollectionCron: "0 */3 * * *"
```

Comment:
This doesn't correspond to the `0 * * * *` you mentioned above

Reply:
Does it need to? It's just an example.

Reply:
You're right